src/BenchmarkDotNet/Engines/EngineEventSource.cs

```diff
@@ -1,12 +1,13 @@
 ﻿using System;
 using System.Diagnostics.Tracing;
 using JetBrains.Annotations;
-
+using System.Diagnostics.Tracing;
 namespace BenchmarkDotNet.Engines
 {
-    [EventSource(Name = "BenchmarkDotNet.EngineEventSource")]
+    [EventSource(Name = EngineEventSource.SourceName)]
     public class EngineEventSource : EventSource
     {
+        const string SourceName = "BenchmarkDotNet.EngineEventSource";
```

Comment:
it looks like there is no need to extract const if it's used only in once place. Could you please revert this change?